modules/core/lib/ImagerySource.js

```diff
@@ -221,12 +221,10 @@ export class ImagerySource {
 
   _localeDateString(s) {
     if (!s) return null;
-    const options = { day: 'numeric', month: 'short', year: 'numeric' };
-    const d = new Date(s);
+    const d = new Date(s + 'T00:00:00Z');  // Add 'T00:00:00Z' to create the date in UTC
     if (isNaN(d.getTime())) return null;
 
-    const localeCode = this.context.systems.l10n.localeCode();
-    return d.toLocaleDateString(localeCode, options);
+    return d.toISOString().split('T')[0];  // Return the date part of the ISO string
```

Comment:
This seems to change what the function does?  
I think what I did before was return display dates formatted for the user's locale?
e.g. "4/17/24" vs "17.04.24"